source/api_reference/index.html.md.erb

```diff
@@ -79,22 +77,22 @@ A successful response includes ``status`` and ``finished`` values:
   </tr>
   <tr>
     <td> submitted </td>
-    <td> Your user has submitted payment details but has not yet selected <b>Confirm</b>. </td>
+    <td> Your user has submitted payment details, and gone through 3D Secure.<br>The PSP has authorised the payment, but the user has not yet selected <b>Confirm</b>. </td>
     <td> false </td>
   </tr>
   <tr>
     <td> capturable </td>
-    <td> The payment is a [delayed capture](/delayed_capture/#delay-taking-a-payment), and your user has submitted payment details and selected <b>Confirm</b>. </td>
+    <td> The payment is a <a href="/delayed_capture/#delay-taking-a-payment" target="blank">delayed capture</a>, and your user has submitted payment details and selected <b>Confirm</b>. </td>
     <td> false </td>
   </tr>
   <tr>
     <td> success </td>
-    <td> Your user successfully completed the payment. </td>
+    <td> Your user successfully completed the payment by selecting <b>Confirm</b>. <br>The user is redirected to an appropriate payment confirmation page on your service.</td>
```

Comment:
do we need to say what would happen with payment links (ie they got the confirmation page that we host?)

Reply:
I think we just remove "on your service" here to account for both situations. If we explain both situations, the table cell gets really long.